homework11/task2.py

```diff
@@ -0,0 +1,71 @@
+# ======== additional tasks=======
+
+# ==== task 3 ====
+class MyStr:
+    def __init__(self, str):
+        self.str = str
+        print(str.upper())
+MyStr("hello")
```

Comment:
Your class does't work like this:
```
my_str = MyStr('test')
print(my_str)
>>> "TEST"
```

To make it works you should override `__str__` method - print uses this method to get a string representation for objects.